coalib/output/ConsoleInteraction.py

```diff
@@ -304,6 +304,7 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        format_args = vars(result)
         try:
             if len(result.affected_code) == 0:
```

Comment:
shouldn't you set `affected_code` to `None` or `Unknown` or something? I think you did that before, sorry for not understanding that earlier^^